### DISCUSSION TITLE
No more charts since Raspi update

### DISCUSSION THREAD

**Discussion by Sayber83** [poblabs/weewx-belchertown#882] [General]
Hi, I have no more charts since apt-get update.
On the debug windows, it shows 
`Uncaught TypeError: Highcharts.chart is not a constructor
    showChart http://belgentier.ddns.net:2000/weewx/belchertown/js/belchertown.js?1682535018:2744
    each jQuery
    showChart http://belgentier.ddns.net:2000/weewx/belchertown/js/belchertown.js?1682535018:1799
    jQuery 8
    showChart http://belgentier.ddns.net:2000/weewx/belchertown/js/belchertown.js?1682535018:1796
    <anonymous> http://belgentier.ddns.net:2000/weewx/belchertown/:147
    jQuery 9
    <anonymous> http://belgentier.ddns.net:2000/weewx/belchertown/:127
    promise callback* http://belgentier.ddns.net:2000/weewx/belchertown/:120
belchertown.js:2744:25
    showChart http://belgentier.ddns.net:2000/weewx/belchertown/js/belchertown.js?1682535018:2744
    each jQuery
    showChart http://belgentier.ddns.net:2000/weewx/belchertown/js/belchertown.js?1682535018:1799
    jQuery 8
    showChart http://belgentier.ddns.net:2000/weewx/belchertown/js/belchertown.js?1682535018:1796
    <anonyme> http://belgentier.ddns.net:2000/weewx/belchertown/:147
    jQuery 9
    <anonyme> http://belgentier.ddns.net:2000/weewx/belchertown/:127
    (Asynchrone : promise callback)
    <anonyme> http://belgentier.ddns.net:2000/weewx/belchertown/:120
`
Any ideas ?
Thanks !

**Comment by ThePeePs** [comment]
see #881 for the fix.

**Comment by Sayber83** [comment]
Thanks and fixed !
